src/transaction/hooks/useWriteContracts.ts

```diff
@@ -8,10 +8,17 @@ type UseWriteContractsParams = {
   setTransactionId: (id: string) => void;
 };
 
-const genericErrorMessage = 'Something went wrong. Please try again.';
+export const genericErrorMessage = 'Something went wrong. Please try again.';
 const uncaughtErrorCode = 'UNCAUGHT_WRITE_TRANSACTIONS_ERROR';
 const errorCode = 'WRITE_TRANSACTIONS_ERROR';
+const eoaErrorMesssage = 'this request method is not supported';
 
+/**
+ * useWriteContracts: Experimental Wagmi hook for batching transactions.
+ * Supports Smart Wallets.
+ * Does not support batch operations and capabilities such as paymasters.
```

Comment:
Is this comment correct?

Reply:
Fixed! Good call on adding comments while the info is "fresh" already getting confused haha